Site/webapp/wdkCustomization/js/client/components/Srt.tsx

```diff
@@ -226,6 +205,7 @@ function SrtForm({
 
   return (
     <form action={formActionUrl} method="post" target="_blank">
+      <button className="btn" type="reset" onClick={onReset}>Reset values to default</button>
```

Comment:
I made this change locally, and it fixed the bug!

```suggestion
      <button className="btn" type="button" onClick={onReset}>Reset values to default</button>
```

Reply:
Good catch